packages/amplify_authenticator/lib/src/blocs/auth/auth_bloc.dart

```diff
@@ -146,6 +146,26 @@ class StateMachineBloc {
     }
   }
 
+  Stream<AuthState> _isValidSession() async* {
+    try {
+      bool isValidSession = await _authService.isValidSession;
```

Comment:
See comments in amplify_auth_service.dart. Making this a getter is a little confusing.
```suggestion
      bool isValidSession = await _authService.isValidSession();
```